include/rocksdb/utilities/transaction.h

```diff
@@ -401,6 +406,9 @@ class Transaction {
 
   virtual bool IsDeadlockDetect() const { return false; }
 
+  // Clear local batch. Should only be called after transaction commits.
+  virtual void ClearBatch() = 0;
```

Comment:
nit:
Since calling Clear() will do more than clear the batch, maybe we should call it `ClearTxn()` or `ClearData()` or `Reset()`
I dont really have a good idea for a good name